frontend/src/components/Conversation/CodeBlock.tsx

```diff
@@ -73,6 +73,8 @@ const formattedCodeOrInitial = (code: string) => {
   }
 };
 
+const chatRouteApi = getRouteApi("/_app/chat/$conversationId");
```

Comment:
Hmmm why do we need this? Nothing like useParams? This is annoying and seemingly unnecessary having to define it in two places, I don't like it.

Reply:
Done